clang/include/clang/CConv/ConstraintVariables.h

```diff
@@ -158,14 +158,11 @@ class ConstraintVariable {
 
   virtual ~ConstraintVariable() {};
 
-  // Sometimes, constraint variables can be produced that are empty. This
-  // tests for the existence of those constraint variables.
-  virtual bool isEmpty(void) const = 0;
```

Comment:
When was this used, before? Perhaps it's been stale for a while?

Reply:
Yep, noticed this a while ago. Figured I'd get rid of it now.